src/docs/transactions.dox

```diff
@@ -54,6 +54,17 @@ discarding any cursor position as well as any key and value.
 
 @snippet ex_all.c transaction commit/rollback
 
+Applications can call WT_SESSION::reset_snapshot to reset snapshots for
+snapshot isolation transactions to update their existing snapshot. It raises
+an error when this API is used for isolation other than snapshot isolation mode
+or when the session has performed any write operations.
+This API internally releases the current snapshot and gets the new running
```

Comment:
How about like this `This API internally releases the current snapshot and gets the new running
transactions snapshot to avoid pinning the content in the database that is no longer needed.`? And also I don't see a need for mentioning the drawback as this function intention itself is to replace the snapshot.

Reply:
The drawback was added to address the @sulabhM review comment.

Reply:
Ok, understood. IMO the statement which you added is not a drawback. How about something like the following?`Applications that don't use read_timestamp for the search may see different results with the updated snapshot compared to earlier`.

Reply:
Thanks @kommiharibabu . This sentence makes more sense.